.github/workflows/e2e-upgrade.yml

```
@@ -169,22 +160,14 @@ jobs:
           fi
 
           bazelisk run //e2e/internal/upgrade:upgrade_test -- --want-worker "$WORKERNODES" --want-control "$CONTROLNODES" --target-image "$IMAGE" "$KUBERNETES_FLAG" "$MICROSERVICES_FLAG"
```

Can't we insert some of these directly into the command?, also `KUBERNETES` <-> `KUBERNETES_FLAG`, same for `MICROSERVICES`

```suggestion
          bazelisk run //e2e/internal/upgrade:upgrade_test -- --want-worker ${{ inputs.workerNodesCount }} --want-control "${{ inputs.controlNodesCount }}" --target-image ${{ inputs.toImage && inputs.toImage || steps.find-image.outputs.output }} ${{ inputs.toKubernetes }} ${{ inputs.toMicroservices }}
```